data/prathamkandari.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Pratham Kandari",
+  "type": "personal",
+  "bio": "**Cloud Enthusiast**",
+
+  "tags": ["WebDev", "DataScience", "OpenSource", "Cloud"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
+    { "icon": "FaGithub", "url": "https://github.com/prathamkandari" },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/pratham-kandari-a02ab3235/"
+    }
+  ],
+
+  "links": [
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/prathamkandari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/<Your Twitter Username>",
+      "icon": "FaTwitter"
+    },
```

Add your Twitter usernames, if not, remove the section.

```suggestion
    {
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/<Your Twitter Username>",
      "icon": "FaTwitter"
    },
```